tests/std/tests/P0896R4_views_drop_while/test.cpp

```diff
@@ -0,0 +1,399 @@
+        }
+
+        STATIC_ASSERT(!CanEnd<const R>);
+    }
+
+    // Validate view_interface::data
+    STATIC_ASSERT(CanMemberData<R> == contiguous_range<V>);
+    STATIC_ASSERT(CanData<R&> == contiguous_range<V>);
+    if constexpr (contiguous_range<V>) {
+        const same_as<remove_reference_t<ranges::range_reference_t<V>>*> auto ptr1 = r.data();
+        assert(to_address(ptr1) == to_address(r.begin()));
+    }
+    STATIC_ASSERT(!CanData<const R&>);
+
+    // Validate view_interface::size
+    STATIC_ASSERT(CanMemberSize<R> == CanSize<Rng>);
+    if constexpr (CanMemberSize<R>) {
+        assert(r.size() == static_cast<decltype(r.size())>(ranges::size(expected)));
+    } else {
+        STATIC_ASSERT(!CanSize<R>);
+    }
+
+    // Validate view_interface::operator[]
+    STATIC_ASSERT(CanIndex<R> == random_access_range<V>);
+    STATIC_ASSERT(!CanIndex<const R>);
+
+    // Validate view_interface::front and back
+    STATIC_ASSERT(CanMemberFront<R> == forward_range<V>);
+    STATIC_ASSERT(CanMemberBack<R> == (bidirectional_range<R> && common_range<R>) );
+    STATIC_ASSERT(!CanMemberFront<const R>);
+    STATIC_ASSERT(!CanMemberBack<const R>);
+
+    if (!is_empty) {
+        if constexpr (forward_range<V>) {
+            assert(r.front() == *begin(expected));
+        }
+
+        if constexpr (CanMemberBack<R>) {
+            assert(r.back() == *prev(end(expected)));
+        }
+    }
+
+    // Validate drop_while_view::base() const&
+    STATIC_ASSERT(CanMemberBase<const R&> == copy_constructible<V>);
+    if constexpr (copy_constructible<V> && forward_range<V>) {
+        same_as<V> auto b1 = as_const(r).base();
+        STATIC_ASSERT(noexcept(as_const(r).base()) == is_nothrow_copy_constructible_v<V>);
+        if (!is_empty) {
+            assert(*b1.begin() == 0); // NB: depends on the test data
+            if constexpr (bidirectional_range<V> && common_range<V>) {
+                assert(*prev(b1.end()) == *prev(end(expected))); // NB: depends on the test data
+            }
+        }
+    }
+
+    // Validate drop_while_view::base() && (NB: do this last since it leaves r moved-from)
+#if !defined(__clang__) && !defined(__EDG__) // TRANSITION, DevCom-1159442
+    (void) 42;
+#endif // TRANSITION, DevCom-1159442
+    if (forward_range<V>) { // intentionally not if constexpr
+        same_as<V> auto b2 = move(r).base();
+        STATIC_ASSERT(noexcept(move(r).base()) == is_nothrow_move_constructible_v<V>);
+        if (!is_empty) {
+            assert(*b2.begin() == 0); // NB: depends on the test data
+            if constexpr (bidirectional_range<V> && common_range<V>) {
+                assert(*prev(b2.end()) == *prev(end(expected))); // NB: depends on the test data
+            }
+        }
+    }
+    return true;
+}
+
+static constexpr int some_ints[]                    = {0, 1, 2, 3, 4, 3, 2, 1};
+static constexpr int only_larger_than_two[]         = {3, 4, 3, 2, 1};

```

Comment:
`only_larger_than_two` is slightly misleading, although I don't really have a great suggestion other than to call it `expected` and remove 381-395. Feel free to make a change or not - it's not a blocker in either case.

Reply:
Renamed to `expected` and `expected_reverse`